### ISSUE TITLE
Fail to build

### ISSUE THREAD

**Issue by hcyrs** [yewstack/yewtify#2]
~/works/yewtify$ cargo build
   Compiling yewtify v0.0.0 (/home/xxxx/works/yewtify)
error: failed to run custom build command for `yewtify v0.0.0 (/home/xxxx/works/yewtify)`

Caused by:
  process didn't exit successfully: `/home/xxxx/works/yewtify/target/debug/build/yewtify-fe819398fe506c0c/build-script-build` (exit code: 1)
--- stderr
Error: environment variable not found

**Comment by paulvt** [comment]
Yeah, `build.rs` uses the `YEWTIFY_OUT` environment variable, but this is not set for a plain `cargo build`.
See also: https://github.com/yewstack/yewtify/blob/master/build.rs#L13

It is set in `examples/yewtify--example/run.sh` but I also cannot get that to work.
